src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/ApplicationPropertyPage.xaml

```diff
@@ -32,20 +32,75 @@
+                HelpUrl="https://go.microsoft.com/fwlink/?linkid=2147236"
+                Category="General">
+    <BoolProperty.DataSource>
+      <DataSource Persistence="ProjectFileWithInterception"
+                  HasConfigurationCondition="False" />
+    </BoolProperty.DataSource>
+    <BoolProperty.Metadata>
+      <NameValuePair Name="DependsOn" Value="TargetFrameworkMoniker;TargetFrameworks" />
+    </BoolProperty.Metadata>
+  </BoolProperty>-->
+
+  <DynamicEnumProperty Name="TargetFramework"
                        DisplayName="Target framework"
                        Description="Specifies the version of .NET that the application targets. This option can have different values depending on which versions of .NET are installed on your computer."
                        HelpUrl="https://go.microsoft.com/fwlink/?linkid=2147236"
                        Category="General"
                        EnumProvider="SupportedTargetFrameworksEnumProvider"
                        MultipleValuesAllowed="False">
-    <DynamicEnumProperty.DataSource>
-      <DataSource PersistedName="TargetFrameworkMoniker"
-                  Persistence="ProjectFileWithInterception"
-                  HasConfigurationCondition="False"/>
-    </DynamicEnumProperty.DataSource>
+    <DynamicEnumProperty.Metadata>
+      <NameValuePair Name="VisibilityCondition">
+        <NameValuePair.Value>(eq (evaluated "Application" "TargetFrameworks") "")</NameValuePair.Value>
+      </NameValuePair>
+    </DynamicEnumProperty.Metadata>
   </DynamicEnumProperty>
 
+  <StringProperty Name="TargetFrameworks"
+                  DisplayName="Target frameworks"
+                  Description="Specifies the semicolon-delimited list of frameworks that this project will target."
+                  HelpUrl="https://go.microsoft.com/fwlink/?linkid=2147236"
+                  Category="General">
+    <StringProperty.DataSource>
+      <DataSource Persistence="ProjectFileWithInterception"
+                  HasConfigurationCondition="False" />
+    </StringProperty.DataSource>
+    <StringProperty.Metadata>
+      <NameValuePair Name="VisibilityCondition">
+        <NameValuePair.Value>false</NameValuePair.Value>
+      </NameValuePair>
+    </StringProperty.Metadata>
+  </StringProperty>
+
+  <StringProperty Name="TargetFrameworksLink"
+                  DisplayName="Target frameworks"
+                  Description="Learn more about targeting multiple frameworks">
+    <StringProperty.DataSource>
+      <DataSource PersistedName="TargetFrameworksLink"
+                  Persistence="ProjectFileWithInterception"
+                  HasConfigurationCondition="False" />
+    </StringProperty.DataSource>
+    <StringProperty.ValueEditors>
+      <ValueEditor EditorType="LinkAction">
+        <ValueEditor.Metadata>
+          <NameValuePair Name="Action" Value="URL" />
+          <NameValuePair Name="URL" Value="https://go.microsoft.com/fwlink/?linkid=2147236" />
```

Comment:
@drewnoakes One thing I found myself wanting was a way to set the link text separate from the title or description. That is, I'd like it if the control showed the title, then the description, then the link with the link text coming from ValueEditor metadata. That may be something to come back to later as it would require more loc support.

Reply:
For the case where we only want to show a hyperlink, using a separate field for that string would require setting an empty display name. I can imagine possible negative knock-on effects from that.